### PR TITLE
Fix kubeadm cidr requirements

### LINKED ISSUE #683: Understand why --pod-network-cidr must match Calico Pool
This doc: http://docs.projectcalico.org/v2.1/getting-started/kubernetes/installation/hosted/kubeadm/
has the statement:
```
The Kubeadm flag --cluster-cidr (pre 1.6) or --pod-network-cidr (1.6+) is not utilized by Calico nor Kubernetes if using Calico for routing.
```
But I'm seeing that if the `--pod-network-cidr` flag is not set to include all calico ippools, then access from pod to a service backed by a pod on a different node does not work when a policy is in place.  Or in simple terms:  the `simple policy demo` in our docs fails intermittently at the "Allow Access using a NetworkPolicy" step.

### PR DESCRIPTION
**Pull request by tmjd**
Fixes #683

I'm checking on the ability to set multiple pools with pod-network-cidr.

### CODE REVIEW

File: master/getting-started/kubernetes/installation/hosted/kubeadm/index.md [hunk 1]
```diff
@@ -62,3 +55,9 @@ $ kubectl get node <master_name> -o yaml | grep kubeadm
 * This install does not configure etcd TLS
 * This install expects that one Kubernetes master node has been labeled with `kubeadm.alpha.kubernetes.io/role: master`
 * This install assumes no other pod network has been installed.
+* The CIDR(s) specified with the flag `--cluster-cidr` (pre 1.6) or
+  `--pod-network-cidr` (1.6+) must match the Calico IP Pools to have Network
+  Policy function correctly. The default is `192.168.0.0/16`.
```
Comment (djosborne) [inline master/getting-started/kubernetes/installation/hosted/kubeadm/index.md]: The default "Calico Pool" is `192.168.0.0/16`

**Comment by djosborne** [comment]
1 thing then LGTM

File: master/getting-started/kubernetes/installation/hosted/kubeadm/index.md (hunk 1 above)
Comment (tmjd) [inline master/getting-started/kubernetes/installation/hosted/kubeadm/index.md]: I was trying to say that the default for those flags is `192.168.0.0/16`, not the Calico IP Pool.

Comment (tmjd) [inline master/getting-started/kubernetes/installation/hosted/kubeadm/index.md]: How about:
The default value for the flag matches the Calico IP Pool of `192.168.0.0/16`

Comment (tmjd) [inline master/getting-started/kubernetes/installation/hosted/kubeadm/index.md]: Or:
The default value for the flag matches the CIDR specified for the Calico IP Pool defined in the manifests linked above.

Comment (djosborne) [inline master/getting-started/kubernetes/installation/hosted/kubeadm/index.md]: Ah. The sentence correctyl insinuates its the default values for those flags. I didn't think that was actulaly true. But if it is, no action. Let me give one final look then ill merge

**Pull request merged**